cddl/measurement-values-map.cddl

```diff
@@ -13,5 +13,6 @@ measurement-values-map = non-empty<{
   ? &(ueid: 9) => ueid-type
   ? &(uuid: 10) => uuid-type
   ? &(name: 11) => text
+  ? &(cryptokeys: 12) => [ + $crypto-key-type-choice ]
```

Comment:
I think making this an array makes things more complicated. Can we make this a singleton?